01/mock-interview-1-submission.md

```diff
@@ -0,0 +1,83 @@
+# Mock Interview #1 Retrospective
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: short-answer
+* id: 1f5bb96e-faa3-428f-b4fe-e1c374d8c960
+* title: Replit Link
+* topics: interview
+* points: 1
+##### !question
+
+Please enter a link to the Replit you used to complete your interview.
+
+##### !end-question
+##### !answer
+
+/.*/
```

Comment:
do we want to alter the regex to make sure they're submitting a replit link?

Reply:
Great question! I'm hesitant to add a regex for this check as I didn't want to limit folx to use replit for their interview. For example, if they were to complete a mock interview with their internship team, they could potentially use an IDE or some internal code editor. I suppose if people code outside replit they can just add their code to a replit and send in that link.

I’d love for this be as little lift as possible for folx so if they would like to submit a link to their code that isn’t a replit link, I’m okay with that as well.

Reply:
Didn't we say that we ask folks use Ada provided questions for this? We could re-evaluate, but if that's the case, then all of their links should be replits